docker/docker-compose.yml

```diff
@@ -1,7 +1,23 @@
 version: '3'
 services:
+  local-storage-service-database:
+    build: ./database
+    restart: always
+    environment:
+      POSTGRES_USER: ${LOCAL_STORAGE_SERVICE_DATABASE_USER}
+      POSTGRES_PASSWORD: ${LOCAL_STORAGE_SERVICE_DATABASE_PASSWORD}
+      POSTGRES_DB: verapdfdb
   local-storage-service:
```

Comment:
add volume mounting here for future container updates without files loss

Reply:
added